modules/apps/frontend-js/frontend-js-bundle-config-extender/src/main/java/com/liferay/frontend/js/bundle/config/extender/internal/servlet/taglib/JSBundleConfigTopHeadDynamicInclude.java

```diff
@@ -87,6 +89,11 @@ public void include(
 							"var MODULE_PATH='", _portal.getPathProxy(),
 							servletContext.getContextPath(), "';"));
 
+					stringWriter.write(
+						StringBundler.concat(
+							"var MODULE_VERSION='", _getModuleVersion(jsConfig),
```

Comment:
Even though I haven't seen it fail in the tests, we should probably test for the existence of `package.json` here and, in case it doesn't exist write:

```javascript
var MODULE_VERSION = undefined;
```

or nothing at all to the HTML. This is because I'm not 100% sure that in every JS bundle (old way of publishing AMD modules) there will be a `package.json` file at that location.

And if there's none, it means it has no bundler support, thus it cannot have "AMD version".

Reply:
@carloslancha :point_up: